Kong/docs.konghq.com
# [DOCU-1634] Docs logo and navbar improvements

### Review
@Kong/team-docs 

### Summary
Proposal for a logo update; logo created by Chuca. Taking team feedback into account, I split the original logo into two images and linked each one to different things (similar to [Netlify Docs](https://docs.netlify.com/)).

**Specific changes:**

* Update and split Kong logo: "Kong" still links to konhq.com, "Docs" links to docs.konghq.com.
* Fix navbar dropdown on mobile - when Docs dropdown is expanded, the text now fully fits in the menu and doesn't float over page text
* Changed link in navbar from "docs.konghq.com" into a relative link to root (`/`). This way, when you build the site locally or in a Netlify preview, the link goes to the homepage of the build, not to the live docs.

**Outstanding issue:** doesn't display very well on the Plugin Hub (padding issues). No easy fix here; the problem is hub.css stlying overwriting navbar-specific styling.

**Out of scope, will be a follow-up ticket**: labelling external links in the navbar as external, and maybe thinking about a better grouping.

### Reason
Cohesiveness, standardization, and a clear sense of where you are in the Kong ecosystem. Makes the navbar change/difference clearer.

https://konghq.atlassian.net/browse/DOCU-1634

### Testing
https://deploy-preview-3108--kongdocs.netlify.app/
* Check out logo and the links to both konghq.com and docs.konghq.com
* Test the "Docs" links next to "Support", should stay in the netlify preview but take you to the homepage
* Resize the page and check logo responsiveness
* At mobile size, check the dropdown menu for "Docs" and make sure that the Contributing link and Get Started button are on a white background
* Check some lower-down pages, eg https://deploy-preview-3108--kongdocs.netlify.app/enterprise/.
